packages/mui-lab/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/lab",
-  "version": "6.0.0-beta.32",
+  "version": "6.0.1-beta.33",
```

Comment:
@DiegoAndai @Janpot can you confirm this version is correct?

Reply:
Wait, it should be `6.0.1-beta.0`, right?

Reply:
Either is fine, I'm fine with 33

Reply:
I don't know, could work.